apps/ledger-live-desktop/src/index.js

```diff
@@ -1,4 +1,4 @@
-if (process.env.PLAYWRIGHT_RUN) {
+if (process.env.PLAYWRIGHT_RUN === "true") {
```

Comment:
we have tons of places where we do `if PLAYWRIGHT_RUN` so i don't understand this solution